script.js

```diff
@@ -1,4 +1,139 @@
+var SCISSORS = "scissors";
+var PAPER = "paper";
+var STONE = "stone";
+var REVERSE_SCISSORS = "reverse scissors";
+var REVERSE_PAPER = "reverse paper";
+var REVERSE_STONE = "reverse stone";
+var numOfUserWins = 0;
+var numOfComputerWins = 0;
+var numOfDraws = 0;
+var numOfRounds = 0;
+var modeAskForUserName = "user types in their name";
+var modeStartPlaying = "sps game starts";
+var gameMode = modeAskForUserName;
+var userName = "";
+
+var programOutput = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var randomNumber = randomInteger + 1;
+  return randomNumber;
+};
+var computersRandomWord = function () {
```

Comment:
As above, best practice is to lead with a verb when creating these function names. See my comment on line 44 to understand why.